js/ui/components/clubhouse.js

```diff
@@ -238,7 +245,17 @@ var ClubhouseNotificationBanner = new Lang.Class({
         else
             this.actor.x = clubhouseWindowX;
 
-        this.actor.x -= this.actor.width + margin;
+        let endX = this.actor.x - this.actor.width - margin;
+
+        if (this._shouldSlideIn)
+            Tweener.addTween(this.actor,
+                             { x: endX,
+                               time: CLUBHOUSE_BANNER_ANIMATION_TIME,
+                               transition: 'easeOutQuad'
+                             });
+        else
+            this.actor.x = endX;
```

Comment:
Nitpick: use curly braces

Reply:
I thought one-lines didn't need curly braces. But I can do that...